test_specs/features/support/API_Objects/category_api.rb

```diff
@@ -0,0 +1,41 @@
+class CategoryAPI
+
+  def initialize(authentication_required, role_user)
+    @apiUtil = APIUtil.new(authentication_required,role_user)
+  end
+
+  # Method to retrieve all the categories of an specific community.
+  # site_name  : Community title to retrieve its categories
+  # header    : Parameters that will be added to URI's headers
+  # params    : Parameters that will be added to URI's body
+  def getAllCategories(site_name, params)
+    communityAPI = CommunityAPI.new(false,nil)
```

Comment:
Maybe we should provide default values to the initializer to avoid passing false and nil